[2022-11-21 22:17:23,502] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: etl_example.transform_data scheduled__2022-11-21T21:00:00+00:00 [queued]>
[2022-11-21 22:17:23,524] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: etl_example.transform_data scheduled__2022-11-21T21:00:00+00:00 [queued]>
[2022-11-21 22:17:23,525] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-11-21 22:17:23,526] {taskinstance.py:1242} INFO - Starting attempt 2 of 2
[2022-11-21 22:17:23,527] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-21 22:17:23,546] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): transform_data> on 2022-11-21 21:00:00+00:00
[2022-11-21 22:17:23,556] {standard_task_runner.py:52} INFO - Started process 1878 to run task
[2022-11-21 22:17:23,560] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'etl_example', 'transform_data', 'scheduled__2022-11-21T21:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/etl_example.py', '--cfg-path', '/tmp/tmp0gx765ud', '--error-file', '/tmp/tmpofx8gnxa']
[2022-11-21 22:17:23,562] {standard_task_runner.py:77} INFO - Job 41: Subtask transform_data
[2022-11-21 22:17:23,622] {logging_mixin.py:109} INFO - Running <TaskInstance: etl_example.transform_data scheduled__2022-11-21T21:00:00+00:00 [running]> on host e6f0f1db9976
[2022-11-21 22:17:23,675] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_example
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-21T21:00:00+00:00
[2022-11-21 22:17:23,718] {python.py:152} INFO - Done. Returned value was: None
[2022-11-21 22:17:23,732] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=etl_example, task_id=transform_data, execution_date=20221121T210000, start_date=20221121T221723, end_date=20221121T221723
[2022-11-21 22:17:23,775] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-21 22:17:23,820] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
